### PR TITLE
Fixes #104: Modifications in the theme

### LINKED ISSUE #104: Modifications in the current UI
Currently, Web and Android versions of susi are following different themes. We should make susi web look alike susi Android.

Please comment your opinions.

Thanks

### PR DESCRIPTION
**Pull request by rishiraj824**
**Fixes issue #104**

**Changes:**
- Lato Font Added
- Added chat bubbles
- New color scheme added for chat bubbles, background
- Placeholder for text input

**Demo Link:**
http://susinewtheme.surge.sh

Screenshots for the change: 
newlayout

### CODE REVIEW

File: src/components/MessageListItem.react.js [hunk 1]
```diff
@@ -56,6 +57,8 @@ class MessageListItem extends React.Component {
     let {message} = this.props;
     let stringWithLinks = this.props.message.text;
     let replacedText = parseAndReplace(stringWithLinks);
+    let messageContainerClasses  = cx('message-container '
```
Comment (AnshulMalik) [inline src/components/MessageListItem.react.js]: Is this cx really needed here ?

Comment (rishiraj824) [inline src/components/MessageListItem.react.js]: @AnshulMalik Yes I couldn't find a way to add multiple classes, please suggest changes :)

Comment (AnshulMalik) [inline src/components/MessageListItem.react.js]: I mean, you are passing a concatenated string to cx, so cx would simple return the same string. 
I'd prefer using cx in more complex scenarios.

By why do you want to add a class from authorName ?

**Comment by rishiraj824** [comment]
@isuruAb Sure I'll add the space and look at the squashing issue. Thanks :)

File: src/components/MessageListItem.react.js (hunk 1 above)
Comment (rishiraj824) [inline src/components/MessageListItem.react.js]: To change the alignment of the messages depending on the author :)

**Comment by uday96** [comment]
Looks great! :smile: 
Add the space as @isuruAb suggested :+1:

File: src/components/MessageListItem.react.js (hunk 1 above)
Comment (AnshulMalik) [inline src/components/MessageListItem.react.js]: But how can we add dynamic style of that class, as we don't know the class name beforehand, it'll be dynamic right ? depending upon the user, or am I going in some other direction?.

**Comment by mariobehling** [comment]
Please resolve merge conflicts.

**Pull request closed without merging**